### PR TITLE
add campaign scenaries

### PR DESCRIPTION
**Pull request by cristianfchq**

### CODE REVIEW

File: src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java [hunk 1]
```diff
@@ -86,4 +122,33 @@ public String getCreatedByText() {
...
+    /**
+     * Gets text fields of workType.
+     *
+     * @return a map with methods of CreatedWorkType
+     */
+    private HashMap<String, StringSupplier> getTxtFields() {
+        webElementAction.clickByLocator(detailsTab);
+        webElementAction.dropDownTillTheEnd();
+        HashMap<String, StringSupplier> mapValues = new HashMap<>();
+//        mapValues.put("Campaign Name", () -> getCampaignNameCreatedText());
```
Comment (JorgeCaceres-Jala) [inline src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java:143]: This comment should be deleted

File: src/main/java/salesforce/ui/pages/lightning/campaign/CreateCampaignPage.java [hunk 2]
```diff
@@ -107,4 +114,35 @@ public CampaignCreatedPage clickSaveBtn() {
...
+     * Saves actions in New work type page in map.
+     *
+     * @param campaignMap is map
+     * @return a map with action of fields
+     */
+    private HashMap<String, VoidSupplier> mapActionsContract(final Map<String, String> campaignMap) {
+        HashMap<String, VoidSupplier> mapActions = new HashMap<>();
+        mapActions.put("Campaign Name", () -> setInputField("Campaign Name", campaignMap.get("Campaign Name")));
+        mapActions.put("Start Date", () -> setInputField("Start Date", campaignMap.get("Start Date")));
+        mapActions.put("End Date", () -> setInputField("End Date", campaignMap.get("End Date")));
```
Comment (JorgeCaceres-Jala) [inline src/main/java/salesforce/ui/pages/lightning/campaign/CreateCampaignPage.java:140]: This lines are less than 120 chars?
Maybe you could run a checkstyle test, I think there some checkstyle corrections missing

File: src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java [hunk 3]
```diff
@@ -27,6 +34,35 @@
...
+    /**
+     * Gets the text inside the element.
+     *
+     * @param field represents the text to be introduced
+     * @return the text set on the field requested contract
+     */
+    public String getTextByFieldName(final String field) {
+        String xpathComplement = XPATH_COMPLEMENTS.get(field);
+        String xpath = String.format(BASE_XPATH, field).concat(xpathComplement);
+        System.out.println("/////////////////////////////////////////////////////");
```
Comment (GustavoZHA) [inline src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java:60]: Delete soups

File: src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java [hunk 4]
```diff
@@ -10,11 +10,18 @@
 
 import org.openqa.selenium.By;
 import salesforce.ui.pages.BasePage;
+import salesforce.utils.strategy.CreatedFeature;
+import salesforce.utils.supplier.StringSupplier;
+
```
[the remark this answers is not in the document]
Comment (cristianfchq) [reply src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java:15]: thanks, It  was solved

File: src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java (hunk 3 above)
Comment (cristianfchq) [reply src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java:60]: ok It was deleted

File: src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java (hunk 1 above)
Comment (cristianfchq) [reply src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java:143]: This comment was deleted

File: src/main/java/salesforce/ui/pages/lightning/campaign/CreateCampaignPage.java (hunk 2 above)
Comment (cristianfchq) [reply src/main/java/salesforce/ui/pages/lightning/campaign/CreateCampaignPage.java:140]: ok

**Review by JorgeCaceres-Jala — APPROVED** [review]

**Pull request merged**